### ISSUE TITLE
Loading two different dictionaries overwrites one another

### ISSUE THREAD

**Issue by paulomann** [MSeal/cython_hunspell#11]
Just want to mention that if you load two different dictionaries using this module, one subscribe one another. For instance, if I run the following code:
```
self.pt_br_dic = Hunspell(pt_BR_path)
self.en_us_dic = Hunspell(en_US_path)
```
It will consider only the `pt_br_dic` and forget the `en_us_dic` one, which means that `self.en_us_dic.suggest` or `self.en_us_dic.spell` will give results as it was the `self.pt_br_dic` variable.

**Comment by MSeal** [comment]
Thanks for the bug report. I'm likely going to do a few fixes here soon and get a new version released.

**Comment by paulomann** [comment]
Thanks!

One more note: the version that I have used is the latest available on pip.

**Comment by MSeal** [comment]
This is fixed in the new 1.3.2 release -- was caching bug.
